package-lock.json

```diff
@@ -1,8 +1,9133 @@
 {
   "name": "mobify-platform-sdks",
   "version": "2.0.0-dev.3",
```

Comment:
This is wrong `2.0.0-dev.3`. It should be `2.0.0-dev.4`.

Usually re-runing the `bump-version` fixes the issue. i.e. We need to run `bump-version` twice.
We may want to create a ticket to investigate why.